eth_tester/backends/pyevm/main.py

```diff
@@ -314,7 +314,8 @@ def revert_to_snapshot(self, snapshot):
     # Meta
     #
     def time_travel(self, to_timestamp):
-        self.chain.header = self.chain.header.copy(timestamp=to_timestamp)
+        self.chain.header = self.chain.header.copy(timestamp=(to_timestamp - 1))
+        self.mine_blocks()
```

Comment:
The assumption here is that for every `time_travel()` we're only mining one block. Do we want to allow an optional arg to customize how many blocks are mined / time travel?

Reply:
Nah, if they want to mine more, they can just `mine_blocks()` again.